Configuration/NodeTypes.Validator.yaml

```diff
@@ -25,17 +25,14 @@
 'Neos.Form.Builder:ValidatorCollection':
   ui:
     icon: 'icon-check-circle'
-    inlineEditable: TRUE
+    label: 'Validators'
+    inlineEditable: true
   options:
     fusion:
       prototypeGenerator: null
   superTypes:
-    'Neos.Neos:ContentCollection': TRUE
+    'Neos.Neos:ContentCollection': true
   constraints:
     nodeTypes:
-      '*': FALSE
+      '*': false
 
-'Neos.Neos:ContentCollection':
-  constraints:
-    nodeTypes:
-      'Neos.Form.Builder:AbstractValidator': FALSE
```

Comment:
These were removed. intentionally?

Reply:
Hey @bwaidelich, they are not removed but gathered in Configuration/NodeTypes.Override.ContentCollection.yaml